### ISSUE TITLE
Unable to Open 'MenuOption' on Mobile Browsers with Long Press

### ISSUE THREAD

**Issue by xdmaury** [silevis/reactgrid#415]
**Describe the bug**
I am experiencing an issue where I cannot open the 'MenuOption' on mobile browsers by long pressing on a cell. This functionality works as expected on desktop browsers when using the right mouse button, but it appears to be non-functional on mobile devices.

**Current behavior**
The 'MenuOption' does not open in mobile browsers when long pressing on a cell.

**Expected behavior**
The 'MenuOption' should open when long pressing on a cell in mobile browsers, similar to the behavior in desktop browsers.

**Screenshots or gifs**
![image](https://github.com/user-attachments/assets/6b7790bc-4ba1-44ff-aca2-b9db9ae9f0b2)
<img width="414" alt="image" src="https://github.com/user-attachments/assets/a0151179-a5e4-405e-b8f1-418e8e1606d9">

**Your environment details**
 - Device:  [iPhone 11/MacBook Pro 14 M1]
 - OS: [iOS 17, macOS 14]
 - Browser [chrome, safari]

Is there an alternative event or method that can be used to open the 'MenuOption' on mobile browsers?

Any guidance or workaround to achieve this functionality on mobile devices would be greatly appreciated. Thank you!

**Comment by webloopbox** [comment]
@xdmaury You're right, the context menu should be activated with a long press. I investigated this, and it seems that the problem lies in event propagation.

**Comment by ecarr-bend** [comment]
Wondering if there is any update on the event propagation issue that is preventing mobile browser long press? Thanks!

**Comment by aleclarsen** [comment]
@xdmaury did you find a work around? Does not look like on v4.1.13 the MenuOption opens with a long press

**Comment by cromatikap** [comment]
Hi @aleclarsen 

The new release contains [a new feature](https://github.com/silevis/reactgrid/pull/461) that was necessary for our project.
@xdmaury and me have decided to not use the `MenuOption` for the moment as our roadmap involves bigger priorities.

Eventually we can help fixing this sooner. What would be the best way to initiate contact with you?

**Comment by xdmaury** [comment]
> @xdmaury did you find a work around? Does not look like on v4.1.13 the MenuOption opens with a long press

Hi, The solution was to create a button to open the same options in menu

<img width="370" alt="image" src="https://github.com/user-attachments/assets/75a1ebce-48c8-4a79-8db4-3ab4282eca85" />
